src/CfgEditor/CfgEditorPanel.ts

```diff
@@ -134,12 +137,27 @@ export class CfgEditorPanel implements vscode.CustomTextEditorProvider {
       'dist',
       'codicon.css',
     ]);
-    const jsUri =
-        getUri(webview, this.context.extensionUri, ['media', 'CfgEditor', 'cfgeditor.js']);
+    const jsUri = getUri(webview, this.context.extensionUri, ['media', 'CfgEditor', 'index.js']);
 
     const cssUri =
         getUri(webview, this.context.extensionUri, ['media', 'CfgEditor', 'cfgeditor.css']);
 
+    const htmlPath =
+        vscode.Uri.joinPath(this.context.extensionUri, 'media/CfgEditor/cfgeditor.html');
+    let html = Buffer.from(await vscode.workspace.fs.readFile(htmlPath)).toString();
+
+    // Apply js and cs to html
+    html = html.replace(/\${nonce}/g, `${nonce}`);
+    html = html.replace(/\${webview.cspSource}/g, `${webview.cspSource}`);
+    html = html.replace(/\${toolkitUri}/g, `${toolkitUri}`);
+    html = html.replace(/\${codiconUri}/g, `${codiconUri}`);
+    html = html.replace(/\${jsUri}/g, `${jsUri}`);
+    html = html.replace(/\${cssUri}/g, `${cssUri}`);
+
+    return html;
+
+    // TODO Remove following unused codes
```

Comment:
How about commenting out the entire code?

Reply:
Aha below `return /*html*/` makes you just leave the comment. Ignore my comment.